lib/api.js

```diff
@@ -62,15 +71,16 @@ export default class AEWeb {
 
       return tx
     })
+
   }
 
   async getRefTransaction(transactions) {
-    const { metaData, refContent } = getMetaData(this.txsContent, transactions, this.metaData, this.sslCertificate)
+    const { metaData, refContent } = getRefTxContent(this.txsContent, transactions, this.metaData, this.sslCertificate)
```

Comment:
There it will crash since the returned value of getRefTxContent is a string and not anymore an object

Reply:
thanks.tested and rebased